### ISSUE TITLE
Allow opting out on a per rule basis

### ISSUE THREAD

**Issue by simonsmith** [postcss/postcss-bem-linter#47]
I think it would be useful to tell bemlint to skip a rule if needed:

``` css
/* bemlint ignore:rule */
.no-flexbox .Component {
  display: block;
}
```

For me the use case is when I need to add some Modernizr fallbacks to a component and have to end up opting out of linting entirely to allow it to work. Moving the rules into a separate file also feels wrong.

**Comment by davidtheclark** [comment]
@simonsmith Can you try master using the comment `/* postcss-bem-linter: ignore */` and see how it goes? If it works, we can close this and cut a new release, I think.

**Comment by simonsmith** [comment]
Works perfectly here. Thanks for this!

**Comment by davidtheclark** [comment]
Great!

**Comment by davidtheclark** [comment]
@necolas What do you think about publishing a new release to add this feature? We'd have to add it to the changelog and docs first. When you transferred the repo to postcss org did you add any new npm owners or would you like to keep that in your court?

**Comment by MoOx** [comment]
https://github.com/postcss/postcss-bem-linter/issues/49#issuecomment-108563651

I think @necolas is not using this module anymore (probably in favor of css-modules), so maybe you should take the lead for this module (and ask to be added as a collab on npm).

**Comment by necolas** [comment]
I added ownership for David. Feel free to add anyone else needed. Some teams at Twitter will still want to use this; I'm moving towards CSS in JS but not really css-modules.

**Comment by davidtheclark** [comment]
Thanks!

**Comment by davidtheclark** [comment]
Released v0.4.0 with this feature.

**Comment by simonsmith** [comment]
:+1:
